core/arch/arm/plat-ls/conf.mk

```diff
@@ -9,13 +9,21 @@ $(call force,CFG_PM_STUBS,y)
 ifeq ($(PLATFORM_FLAVOR),ls1021atwr)
 include core/arch/arm/cpu/cortex-a7.mk
 CFG_TEE_CORE_NB_CORE = 2
+CFG_TZDRAM_START ?= 0xbc000000
+CFG_TZDRAM_SIZE ?= 0x03e00000
+CFG_SHMEM_START ?= 0xbfe00000
+CFG_SHMEM_SIZE ?= 0x00100000
 CFG_BOOT_SYNC_CPU ?= y
 CFG_BOOT_SECONDARY_REQUEST ?= y
 endif
 
 ifeq ($(PLATFORM_FLAVOR),ls1021aqds)
 include core/arch/arm/cpu/cortex-a7.mk
 CFG_TEE_CORE_NB_CORE = 2
+CFG_TZDRAM_START ?= 0xfc000000
+CFG_TZDRAM_SIZE ?= 0x03f00000
+CFG_SHMEM_START ?= 0xfff00000
```

Comment:
Please keep following macros as per earlier memory map only.
CFG_TZDRAM_SIZE ?= 0x03e00000
CFG_SHMEM_START ?= 0xffe00000

Reply:
Thanks for the review.
I will update **conf.mk** according to your comment but please cross check with previous implementation: dram size was [63MByte](https://github.com/OP-TEE/optee_os/pull/2284/files/96225f85889e4897adea2062ee91f47d43bcd417#diff-59a18e482b970cabae42da01665ead99L73) and SHM was [the last DRAM 1MByte](https://github.com/OP-TEE/optee_os/pull/2284/files/96225f85889e4897adea2062ee91f47d43bcd417#diff-59a18e482b970cabae42da01665ead99L75).

Reply:
Following is memory map on ls1021aqds:
TEE + TA RAM size = 62MB
NSEC Shared memory = 1MB
Total DRAM for OP-TEE = 63MB

Actually this is not 64MB due to NSEC shared memory needs to be on 2MB aligned boundary for TZASC 380 configuration.

Reply:
i see, my mistake. thanks. i have pushed a P-R for the change in **plat-ls**. @b49020 @pangupta, can you provide a tag (or comment) on #2322?